tests/workers/task.yaml

```diff
@@ -0,0 +1,12 @@
+summary: Test the MATCH function.
+
+prepare: |
+    if [ ! -f .spread-reuse.yaml ]; then
+        touch /run/spread-reuse.yaml
+        ln -s /run/spread-reuse.yaml .spread-reuse.yaml
+    fi
+
+execute: |
+    # Check it is possible to use just 1 worker
+    spread -single-worker lxd:ubuntu-16.04:checks/ &> task1.out
+    [ "$(grep -c Allocating task1.out)" -eq 1 ]
```

Comment:
Can we have two tests here ? One that checks that the "grep -c" count is one with single worker and one that it is greater than 1 without single-worker? To ensure the option is really having an effect?

Reply:
Nice catch, but it is not possible yet, I'll implement this test once we can use the google backend for the tests because with lxd we always use 1 worker.  
The pr with the change to start using gitbuh actions (#107) will allow to test that.

Reply:
Sure, thanks